actions/send-sves.vbs

```diff
@@ -44,62 +44,60 @@ changelog = array()
   CheckBox 5, 65, 190, 10, "Check here to submit a query for covered quarters only.", quarters_check
-  OptionGroup RadioGroup1
-    RadioButton 190, 15, 60, 15, "SSN? (default)", SSN_radiobutton
-    RadioButton 190, 30, 75, 15, "Claim # on UNEA?", UNEA_radiobutton
-    RadioButton 190, 45, 75, 15, "Claim # on BNDX?", BNDX_radiobutton
+  EditBox 70, 80, 125, 15, worker_signature
   ButtonGroup ButtonPressed
-    OkButton 160, 80, 50, 15
-    CancelButton 215, 80, 50, 15
-  EditBox 80, 80, 70, 15, worker_signature
-  Text 5, 30, 120, 10, "Enter your member number (ex: 01): "
-  Text 5, 85, 70, 10, "Sign your case note:"
-  GroupBox 185, 5, 80, 55, "Number to use?"
-  Text 5, 10, 80, 10, "Enter your case number:"
+    PushButton 130, 5, 65, 15, "SVES TE02.12.13", POLI_TEMP_SVES_button
+    OkButton 100, 100, 45, 15
+    CancelButton 150, 100, 45, 15
+  Text 5, 50, 60, 10, "Requested Using:"
+  Text 5, 85, 60, 10, "Worker Signature:"
+  Text 5, 10, 50, 10, "Case Number:"
+  Text 5, 30, 55, 10, "MEMB Number:"
 EndDialog
 
 Do
     Do
         err_msg = ""
-        Dialog Dialog1  					'Calling a dialog without a assigned variable will call the most recently defined dialog
-        cancel_without_confirmation
-
-        member_number = trim(member_number)
-        call validate_MAXIS_case_number(err_msg, "*")
-        If member_number = "" Then err_msg = err_msg & vbNewLine & "* Enter the member number that needs a SVES sent."
-
-        If err_msg <> "" Then MsgBox "*** Please Resolve to Continue: " & vbNewLine & err_msg
+        DIALOG Dialog1  					'Calling a dialog without a assigned variable will call the most recently defined dialog
+		cancel_confirmation
+		IF MAXIS_case_number = "" OR (MAXIS_case_number <> "" AND len(MAXIS_case_number) > 8) OR (MAXIS_case_number <> "" AND IsNumeric(MAXIS_case_number) = False) THEN err_msg = err_msg & vbCr & "Please enter a valid case number."
+		If trim(member_number) = "" Then err_msg = err_msg & vbNewLine & "Please enter the member number that needs a SVES sent."
+		IF SVES_actions = "Select One:" THEN  err_msg = err_msg & vbnewline & "Please select the number you want to use to initate the query."
+		IF ButtonPressed = POLI_TEMP_SVES_button THEN CALL view_poli_temp("02", "12", "13", "") 'TE02.12.13 SVES TPQY INTERFACE
+        If err_msg <> "" Then MsgBox "*** Resolve to Continue: " & vbNewLine & err_msg
     Loop until err_msg = ""
     Call check_for_password(are_we_passworded_out)
 Loop until are_we_passworded_out = FALSE
 
 'Goes to MEMB to get info
-call navigate_to_MAXIS_screen("stat", "memb")
+CALL navigate_to_MAXIS_screen_review_PRIV("STAT", "MEMB", is_this_priv)
+IF is_this_priv = TRUE THEN script_end_procedure("This case is privileged, the script will now end.")
 
 'Goes to the right HH member
 EMWriteScreen member_number, 20, 76 'It does this to make sure that it navigates to the right HH member.
```

Comment:
Line 108 - Add information about the script ending here. It sounds like an err_msg.

Reply:
updated